frontend/src/views/web/challenge/participate.html

```diff
@@ -28,7 +28,11 @@
                                                 <a ng-class="challenge.isPrev" class="btn-floating btn-pagination waves-effect waves-light " ng-click="challenge.load(challenge.existTeam.previous)">
                                                     <i class="fa fa-chevron-left"></i>
                                                 </a>
-                                                <span class="pagination-title"> <strong class="text-med-black"> Page {{challenge.currentPage | ceil}} of {{challenge.existTeam.count/10 | ceil}} </strong></span>
+                                                <span class="pagination-title">
+                                                    <strong class="text-med-black" ng-if="!challenge.existTeam.results"> Page 1 of {{challenge.existTeam.count/10 | ceil}} </strong>
```

Comment:
I think so you shouldn't hardcore this here, you could make changes in the `js` file.

Reply:
@gautamjajoo done.